Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Core tools for building Linaro's static web sites.
 
-# Set the base image to Ubuntu (version 18.04) Uses the new "ubuntu-minimal"
-# image Should be Alpine like all the cool kids, but
+# Set the base image to Ubuntu (version 18.04).
```

Comment:
I like, but remove the trailing period, or add to all other lines. :smile_cat:

Reply:
Done ... although you were responsible for most of those missing!